Docs/development/jasp-build-guide-linux.md

```diff
@@ -0,0 +1,52 @@
+# Linux Build Guide
+
+If you have not cloned the `jasp-desktop` repository, please head back to the build introduction, and after cloning the repo, and updating the submodules continue with the rest of this article.
```

Comment:
`the build introduction` -> `the [build introduction](link to that introduction)`